components/WelcomeAttentionMsg/WelcomeAttentionMsg.js

```diff
@@ -15,9 +15,12 @@ export default function WelcomeAttentionMsg() {
 
 const Wrapper = styled.div`
   display: flex;
-  flex-direction: column;
+  flex-direction: flex;

```

Comment:
Do you mean `flex-direction: row;` here?

Reply:
🤦 thx. changed.